### PR TITLE
New redeem

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +49,19 @@ A rinkeby example for the DAO was deployed, with its Avatar on address [0x5B3dd2
...
+## Redemption Script
+
+In order to run the redeeming script, start by adding the following arguments to your `.env` file.
+
+```bash
+private_key=YOUR_PRIVATE_KEY
+ws_provider=YOUR_WEB_SOCKER_WEB3_PROVIDER
+from_block=THE_BLCOK_TO_START_SCANNING_FROM
+gas_price=GAS_PRICE_FOR_TXS
+NectarReputationRedeemerAddress=REDEEMING_CONTRACT_ADDRESS
```
Comment (orenyodfat) [inline README.md]: this is the batcher contract ?
we can keep its address hard coded .no?

File: contracts/Migrations.sol [hunk 2]
```diff
@@ -0,0 +1,23 @@
+pragma solidity >=0.4.21 <0.6.0;
```
Comment (orenyodfat) [inline contracts/Migrations.sol]: how about using this package/contract ?
https://github.com/daostack/web3-transaction-batcher/blob/master/contracts/Batcher.sol#L4

File: README.md (hunk 1 above)
Comment (ben-kaufman) [reply README.md]: Well we can but then it will work only on mainnet.

Comment (orenyodfat) [reply README.md]: we can have it also on Rinkeby

**Comment by orenyodfat** [comment]
we need to have the batcher redeemer limitation.
what is the max number of batched tx we can do in one tx ?
then support that on the script .(batch it to max number of txs)

**Comment by orenyodfat** [comment]
ganache test

File: contracts/NectarReputationRedeemer.sol [hunk 3]
```diff
@@ -0,0 +1,25 @@
+pragma solidity >=0.4.21 <0.6.0;
```
Comment (orenyodfat) [inline contracts/NectarReputationRedeemer.sol]: how about fix solc version to 0.5.13 ?

**Comment by orenyodfat** [comment]
please add travis test

File: package.json [hunk 4]
```diff
@@ -4,12 +4,13 @@
   "description": "nectar dao deployment repo",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
     "migrate_prod": "npm run prepare_prod && daostack-migrate dao --prev-migration node_modules/@daostack/migration/migration.json --gasPrice 20 --params ./params.json --custom-abis-location ./custom-abis --arc-version 0.0.1-rc.33",
-    "migrate": "npm run prepare && daostack-migrate dao --prev-migration node_modules/@daostack/migration/migration.json --gasPrice 20 --params ./params.json --custom-abis-location ./custom-abis --arc-version 0.0.1-rc.33",
     "prepare": "node ./prepare.js ./parameters/deployment.json ./parameters/mainnet_params_25_11.json",
-    "prepare_prod": "node ./prepare.js ./parameters/deployment_prod.json ./parameters/production_mainnet_params.json",
-    "redeem": "node -e 'require(\"./scripts/redeemRep.js\").startRedeem()'"
+    "migrate": "npm run prepare && daostack-migrate dao --prev-migration node_modules/@daostack/migration/migration.json --gasPrice 20 --params ./params.json --custom-abis-location ./custom-abis --arc-version 0.0.1-rc.33",
```
Comment (orenyodfat) [inline package.json]: please use arc 34

**Pull request merged**